src/main/java/mods/railcraft/common/blocks/aesthetics/brick/BrickTheme.java

```diff
@@ -135,6 +146,85 @@ protected void initVariant(BlockBrick block, BrickVariant variant) {
             if (variant != BrickVariant.BRICK)
                 super.initVariant(block, variant);
         }
+
+
+    },
+    REDNETHER(RailcraftBlocks.BRICK_RED_NETHER, MapColor.NETHERRACK) {
+        @Override
+        public ItemStack getStack(int qty, @Nullable IVariantEnum variant) {
+            if (variant == BrickVariant.BRICK)
+                return new ItemStack(Blocks.RED_NETHER_BRICK, qty);
+            return super.getStack(qty, variant);
+        }
+
+        @Nullable
+        @Override
+        public ItemStack getStack(int qty, int meta) {
+            if (BrickVariant.fromOrdinal(meta) == BrickVariant.BRICK)
+                return new ItemStack(Blocks.RED_NETHER_BRICK, qty);
+            return super.getStack(qty, meta);
+        }
+
+        @Override
+        public void initRecipes(BlockBrick block) {
+            CraftingPlugin.addFurnaceRecipe(new ItemStack(Blocks.RED_NETHER_BRICK), getStack(1, BLOCK), 0);
+        }
+
+        @Override
+        protected void initVariant(BlockBrick block, BrickVariant variant) {
```

Comment:
Check the Nether Brick Code. The only reason those variants need an override is to ensure that they can be crafted despite the fact you pick up/craft the brick variant rather than the block variant.